app/models/spotlight/exhibit.rb

```diff
@@ -85,6 +86,10 @@ def searchable?
       blacklight_config.search_fields.any? { |_k, v| v.enabled && v.include_in_simple_select != false }
     end
 
+    def set_default_thumbnail
+      self.thumbnail ||= searches.first.try(:thumbnail)
```

Comment:
Is this legit to just set the thumbnail as the search's thumbnail?  Should we dup/clone it instead?